spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -144,60 +155,71 @@ def setup(self, mute_inline_plotting=None, show_plot_outline=None,
 
         copyfig_btn = create_toolbutton(
             self, icon=ima.icon('editcopy'),
-            tip=(_("Copy plot to clipboard as image (%s)") %
-                 CONF.get_shortcut('plots', 'copy')),
+            tip=_("Copy plot to clipboard as image"),
             triggered=self.copy_figure)
+        copyfig_btn.shortcut_data = ('plots', 'copy')
+        copyfig_btn.installEventFilter(self)
 
-        closefig_btn = create_toolbutton(
-                self, icon=ima.icon('editclear'),
-                tip=_("Remove image"),
-                triggered=self.close_figure)
+        self.closefig_btn = create_toolbutton(
+            self, icon=ima.icon('editclear'),
+            tip=_("Remove plot"),
+            triggered=self.close_figure)
+        self.closefig_btn.shortcut_data = ('plots', 'close')
+        self.closefig_btn.installEventFilter(self)
 
         closeall_btn = create_toolbutton(
-                self, icon=ima.icon('filecloseall'),
-                tip=_("Remove all images from the explorer"),
-                triggered=self.close_all_figures)
+            self, icon=ima.icon('filecloseall'),
+            tip=_("Remove all plots"),
+            triggered=self.close_all_figures)
+        closeall_btn.shortcut_data = ('plots', 'close all')
+        closeall_btn.installEventFilter(self)
 
-        vsep1 = QFrame()
-        vsep1.setFrameStyle(53)
+        vertical_separator1 = QFrame()
+        vertical_separator1.setFrameStyle(53)
 
         goback_btn = create_toolbutton(
-                self, icon=ima.icon('ArrowBack'),
-                tip=_("Previous figure ({})").format(
-                      CONF.get_shortcut('plots', 'previous figure')),
-                triggered=self.go_previous_thumbnail)
+            self, icon=ima.icon('ArrowBack'),
+            tip=_("Previous plot"),
+            triggered=self.go_previous_thumbnail)
+        goback_btn.shortcut_data = ('plots', 'previous figure')
+        goback_btn.installEventFilter(self)
 
         gonext_btn = create_toolbutton(
-                self, icon=ima.icon('ArrowForward'),
-                tip=_("Next figure ({})").format(
-                      CONF.get_shortcut('plots', 'next figure')),
-                triggered=self.go_next_thumbnail)
+            self, icon=ima.icon('ArrowForward'),
+            tip=_("Next plot"),
+            triggered=self.go_next_thumbnail)
+        gonext_btn.shortcut_data = ('plots', 'next figure')
+        gonext_btn.installEventFilter(self)
 
-        vsep2 = QFrame()
-        vsep2.setFrameStyle(53)
+        vertical_separator2 = QFrame()
+        vertical_separator2.setFrameStyle(53)
 
         self.zoom_out_btn = create_toolbutton(
-                self, icon=ima.icon('zoom_out'),
-                tip=_("Zoom out (Ctrl + mouse-wheel-down)"),
-                triggered=self.zoom_out)
+            self, icon=ima.icon('zoom_out'),
+            tip=_("Zoom out (Ctrl + mouse-wheel-down)"),
```

Comment:
I think we need to add here something to take OSX users into account where this would be a `Cmd`instead of a `Ctrl.

Is this the case @impact27 (I do not have a mouse so I am not sure)

Reply:
I think this is out of scope for this PR and should be reported in another issue and addressed in another PR.

Reply:
Yes usually Ctrl and Cmd are swapped on OSX.

Reply:
I got an idea, I'm going to fix this here afterall.